doc/source/user_guide/data_sources_tutorial.rst

```diff
@@ -0,0 +1,310 @@
+.. _user_guide_data_sources:
+
+===============
+DPF DataSources
+===============
+
+The DataSources operator is a class available as a submodule of the 'ansys.dpf.core'
+package. It contains files with the analysis results. So this object is used in
+the first basic step: define simulation data, by defining where it is located.
+
+This example will describe the functionalities and functions of the class DataSources
+and can be generalised for all the files formats accepted by Py-DPF:
+
+.. _list_extensions_solvers:
+**Supported apps by DPF and their related formats**
+
+
+   - MAPDL files: .rst, .mode, .rfrq, .rdsp
+   - LS-DYNA files: .d3plot, .binout
+   - Fluent files: CFF restart files(.cas/dat.h5) and Project files (.flprj)
+   - CFX files: CFF files(.cas/dat.cff.res) and Project files (.flprj)
+
+DataSources creation
+--------------------
+
+**a) Class** :class:`DataSources <ansys.dpf.core.data_sources.DataSources>`
+If you are sure that your file has exactly the extensions keys listed above,
+the DataSources object can be created by directly calling the class with the
+file path as an argument :
+
+.. code-block:: python
+
+    from ansys.dpf import core as dpf
+    # Create the DataSources object containing the given file
```

Comment:
```suggestion
    # Create a DataSources for a single given file
```

Reply:
Create a DataSources object for a single file

Reply:
```suggestion
    # Create a DataSources object for a single file
```